### PR TITLE
GENIE add job definition

### PR DESCRIPTION
**Pull request by thomasyu888**

### CODE REVIEW

File: config/prod/genie.yaml [hunk 1]
```diff
@@ -5,4 +5,5 @@ dependencies:
   - sandcastlevpc
 parameters:
   ComputeImageId: ami-283da057
-  ContainerImage: sagebionetworks/genie:validation
+  ContainerImage: sagebionetworks/genie:latest
+  ContainerImage2: sagebionetworks/genie:vcf2maf
```
Comment (zaro0508) [inline config/prod/genie.yaml]: To do this you'll need to add `ContainerImage2` to the parameters section in `templates/genie.yaml`

Comment (thomasyu888) [reply config/prod/genie.yaml]: Added

File: templates/genie.yaml [hunk 2]
```diff
@@ -140,6 +143,56 @@ Resources:
           - ContainerPath: "/root/.ssh/genie.pem"
             ReadOnly: false
             SourceVolume: geniePem
+  BatchJobDefinition3:
```
Comment (zaro0508) [inline templates/genie.yaml]: how about `MafBatchJob`?

File: templates/genie.yaml [hunk 3]
```diff
@@ -7,7 +7,10 @@ Parameters:
     Description: The Genie compute resource image ID
     Type: 'AWS::EC2::Image::Id'
   ContainerImage:
-    Description: The Genie docker container image
+    Description: The Genie main processing and validation docker container image
+    Type: String
+  ContainerImage2:
```
Comment (zaro0508) [inline templates/genie.yaml]: might be nice if you could give these more descriptive names. how about MutationImage?

File: templates/genie.yaml [hunk 4]
```diff
@@ -292,6 +345,6 @@ Outputs:
     Export:
       Name: !Sub '${AWS::Region}-${AWS::StackName}-BatchJobQueue'
   BatchJobDefinition:
```
Comment (thomasyu888) [inline templates/genie.yaml]: This name isn't tied to the "BatchJobDefinition" from above is it?  Or does this also need to be "ValidationBatchJob"

Comment (zaro0508) [reply templates/genie.yaml]: Export resource name doesn't have to match however it is a good practice to keep it the same.  You'll want to export info for all 3 batch job definitions here.  

```
  ValidationBatchJob:
    Value: !Ref ValidationBatchJob
    Export:
      Name: !Sub '${AWS::Region}-${AWS::StackName}-ValidationBatchJob'
  MainProcessingBatchJob:
    Value: !Ref MainProcessingBatchJob
    Export:
      Name: !Sub '${AWS::Region}-${AWS::StackName}-MainProcessingBatchJob'
  MafBatchJob:
    Value: !Ref MafBatchJob
    Export:
      Name: !Sub '${AWS::Region}-${AWS::StackName}-MafBatchJob'
```

File: config/prod/genie.yaml [hunk 5]
```diff
@@ -5,4 +5,5 @@ dependencies:
   - sandcastlevpc
 parameters:
   ComputeImageId: ami-283da057
-  ContainerImage: sagebionetworks/genie:validation
+  ContainerImage: sagebionetworks/genie:latest
```
Comment (zaro0508) [inline config/prod/genie.yaml]: You'll need to update these to match properties in templates/genie.yaml

```
MainImage: sagebionetworks/genie:latest
MutationImage: sagebionetworks/genie:vcf2maf
```

**Pull request merged**